### ISSUE TITLE
Use geoJSON for data file format

### ISSUE THREAD

**Issue by jones139** [jones139/BrewMap#26]
The data files use a made-up json format.   We should use geoJSON (http://geojson.org/)  for future compatibility with other software.
This is not a particular issue for the BrewMap, but if we use this code for other things that may need linear features or areas rather than just points, a geoJSON compatible format would be sensible.  (I am thinking about the uk historic map, which will want to show roman roads etc as linear features.

**Comment by craigloftus** [comment]
It should be relatively easy to change the format. It seems like we have a pretty simple FeatureCollection.

Perhaps we should use the opportunity to also change the structure of data and queries currently used? The present structure isn't very scalable.

An example off the top of my head would be for the client to request a number of entities within the bounding box of the map? The server would respond with a random set of matches of all types, up to the limit specified by the client. The client could further specify the types of entities desired in the query. The problem with this example is that it would be more expensive for the server.

**Comment by jones139** [comment]
You are right, but I think we should treat it as two different tasks - converting the server and client sides to use geoJSON will be quite easy (I have found out that postgis will output geoJSON).  I will open a separate issue to discuss how to deal with scalability.
